packages/input/src/LionInput.js

```diff
@@ -43,6 +47,9 @@ export class LionInput extends LionField {
         } else if (this.hasAttribute('value')) {
           native.setAttribute('value', this.getAttribute('value'));
         }
+        if (this.placeholder) {
```

Comment:
I didn't notice this earlier, but putting the logic in the slots getter means this only is synced when the SlotMixin is connected, which happens on connectedCallback of the host element. If you change it later, it is not synced.

Usually for syncing attributes to the native node, we do it in the `updated` callback, you can follow the example of `type` and `step` in LionInput for this. Perhaps good to add to your test that it should sync the placeholder attribute to the native node when changing it after initialisation of the component.

Reply:
@jorenbroekema,

Ohhh, that is so cool! Well spotted!
Learned something new today :)

Oh, I also fixed it...